README.md

```diff
@@ -7,21 +7,25 @@
 
 Official explorer for the Oasis Network.
 
-:warning: **Oasis Explorer is currently in development and is not available yet.**
+:warning:
+**Oasis Explorer is currently in development and is not available yet.**
 
 ## Oasis Indexer API
 
-Deployed version is at https://index.oasislabs.com/v1/ with OpenAPI spec at https://index.oasislabs.com/v1/spec/v1.html (and https://index.oasislabs.com/v1/spec/v1.yaml).
+Deployed version is at <https://index.oasislabs.com/v1/> with OpenAPI
+[HTML][indexer-html-spec] and [YAML][indexer-yaml-spec] specs.
 
 <details>
 <summary>
 Bleeding-edge API version
 </summary>
 
-OpenAPI spec for the bleeding-edge version is at https://github.com/oasisprotocol/oasis-indexer/blob/main/api/spec/v1.yaml.
+OpenAPI spec for the [bleeding-edge version][indexer-bleeding-edge-spec].
 
-To quickly run latest emerald indexer locally without running a node, replace https://github.com/oasisprotocol/oasis-indexer/blob/d48de37/tests/e2e/config/e2e-dev.yml#L1-L8 with:
+To quickly run latest emerald indexer locally without running a node,
+[replace following lines][replace-lines-to-run-latest-emerald] with:
 
+<!-- markdownlint-disable MD013 -->
```

Comment:
Similarly, I would prefer human readable names in the ignores so it is more obvious what they do and why they are there:

```suggestion
<!-- markdownlint-disable line-length -->
```